resources/commands.json

```diff
@@ -282,6 +307,94 @@
 						"default": "algorithm",
 						"hidden": true,
 						"description": "The type of the asset to register"
+					},
+					{
+						"name": "subscriptionNFT",
+						"type": "string",
+						"default": "",
+						"description": "The address of the subscription NFT used to protect the access to the asset"
+					},					
+					{
+						"name": "nftType",
+						"type": "number",
+						"default": "721",
+						"hidden": true,
+						"description": "The NFT type"
+					}					
+				]
+			}, {
+				"name": "register-service",
+				"description": "Register a new web service",
+				"details": "This command registers a new web service on the Nevermined network. The service can be the representation of any regular Web Service (REST, JSON-RPC, etc).",
+				"examples": ["ncli assets register-service --name 'My AI Bot' --author 'John Doe' --price 0 --language python --entrypoint 'python word_count.py' --container python:3.8-alpine --urls https://www.apache.org/licenses/LICENSE-2.0 --contentType text/plain"],
+				"commandHandler": "registerAsset",
+				"optionalArguments": [{
+						"name": "name",
+						"type": "string",
+						"demandOption": true,
+						"description": "The asset name"
+					},
+					{
+						"name": "author",
+						"type": "string",
+						"demandOption": true,
+						"description": "The author of the asset"
+					},
+					{
+						"name": "dateCreated",
+						"type": "string",
+						"description": "When the asset was created"
+					},
+					{
+						"name": "access",
+						"type": "string",						
+						"default": "direct",
+						"description": "The type of access control associated to the asset. If `direct` the asset will allow direct payment, if `susbcription` it will allow access via NFT susbcription, if both it will configure both methods."
```

Comment:
do web services work without an nft subscription?

Reply:
fixed to get a default option that is hidden